internal/pkg/cli/pipeline_override.go

```diff
@@ -0,0 +1,155 @@
+	}
+
+	prompt := prompt.New()
+
+	cmd := &overridePipelineOpts{
+		overrideOpts: &overrideOpts{
+			overrideVars: vars,
+			fs:           fs,
+			cfgStore:     config.NewSSMStore(identity.New(defaultSess), ssm.New(defaultSess), aws.StringValue(defaultSess.Config.Region)),
+			prompt:       prompt,
+			cfnPrompt:    selector.NewCFNSelector(prompt),
+			spinner:      termprogress.NewSpinner(log.DiagnosticWriter),
+		},
+		ws:       ws,
+		wsPrompt: selector.NewWsPipelineSelector(prompt, ws),
+	}
+	cmd.overrideOpts.packageCmd = cmd.newPipelinePackageCmd
+	return cmd, nil
+}
+
+// Validate returns an error for any invalid optional flags.
+func (o *overridePipelineOpts) Validate() error {
+	if err := o.overrideOpts.Validate(); err != nil {
+		return err
+	}
+	return o.validatePipelineName()
+}
+
+// Ask prompts for and validates any required flags.
+func (o *overridePipelineOpts) Ask() error {
+	if o.name == "" {
+		if err := o.askPipelineName(); err != nil {
+			return err
+		}
+	}
+	return o.overrideOpts.Ask()
+}
+
+// Execute writes IaC override files to the local workspace.
+func (o *overridePipelineOpts) Execute() error {
+	o.overrideOpts.dir = func() string {
+		return o.ws.PipelineOverridesPath(o.name)
+	}
+	return o.overrideOpts.Execute()
+
+}
+
+func (o *overridePipelineOpts) validatePipelineName() error {
+	if o.name == "" {
+		return nil
+	}
+	pipelines, err := o.ws.ListPipelines()
+	if err != nil {
+		return fmt.Errorf("list pipelines in the workspace: %v", err)
+	}
+	var names []string
+	for _, pipeline := range pipelines {
+		names = append(names, pipeline.Name)
+	}
+	if !contains(o.name, names) {
+		return fmt.Errorf("pipeline %q does not exist in the workspace", o.name)
+	}
+	return nil
+}
+
+func (o *overridePipelineOpts) askPipelineName() error {
+	pipeline, err := o.wsPrompt.WsPipeline("Which pipeline's resources would you like to override?", "")
+	fmt.Printf("The pipeline which you want to override is %v", pipeline.Name)
+	if err != nil {
+		return fmt.Errorf("select pipeline name from workspace: %v", err)
+	}
+	o.name = pipeline.Name
+	return nil
+}
+
+func (o *overridePipelineOpts) newPipelinePackageCmd(tplBuf stringWriteCloser) (executor, error) {
+	cmd, err := newPackagePipelineOpts(packagePipelineVars{
+		name:    o.name,
+		appName: o.appName,
+	})
+	if err != nil {
+		return nil, err
+	}
+	cmd.tmplWriter = tplBuf
+	return cmd, nil
+}
+
+func buildPipelineOverrideCmd() *cobra.Command {
+	vars := overrideVars{}
+	cmd := &cobra.Command{
+		Use:   "override",
+		Short: "Override the AWS CloudFormation template of a Pipeline.",
+		Long: `Scaffold Infrastructure as Code patch files. 
+Customize the patch files to change resource properties, delete 
+or add new resources to the Pipeline's AWS CloudFormation template.`,
+		Example: `
+  Create a new Cloud Development Kit application to override the "frontend" pipeline template.
+  /code $ copilot pipeline override -n frontend -e test --toolkit cdk`,
```

Comment:
```suggestion
  Create a new Cloud Development Kit application to override the "myrepo-main" pipeline template.
  /code $ copilot pipeline override -n myrepo-main --toolkit cdk`,
```
two small things here:
- removing the `-e` flag that was for environment
- updating the pipeline name to match closer to what we default name the pipelines